### ISSUE TITLE
Unable to build Netopeer2-server as executable in QNX

### ISSUE THREAD

**Issue by SumukhNiketan** [CESNET/netopeer2#489]
Hi,

I have built Netopeer2-server downloading from Github in QNX environment.

I have not tried building Netopeer2-server in Linux/Unix environment.

The built binary in QNX is a shared Library. Is it what it is intended to be in UNIX/LINUX or it is an executable/binary.

 When I run the Netopeer2-server I am getting the below error.
"ksh: ./netopeer2-server: Attempting to exec a shared lib"

**Comment by michalvasko** [comment]
Hi,
*netopeer2* is an executable that needs *libnetconf2* library, are you not able to use CMake for building on QNX?

Regards,
Michal

**Comment by SumukhNiketan** [comment]
I was able to build Libnetconf2 using cmake. When I do a "file" on the Netopeer2-server it says its a shared library as below.

netopeer2-server: ELF 32-bit MSB shared object, PowerPC or cisco 4500, version 1 (SYSV), dynamically linked, with unknown capability 0x41000000 = 0xf676e75, with unknown capability 0x10000 = 0x70401, not stripped

I had faced some compilation issues while compiling Netopeer2-server which I had resolved which were specific to QNX.
